mypy/checkexpr.py

```diff
@@ -273,6 +273,16 @@ def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
                         not tp.type_object().runtime_protocol):
                     self.chk.fail('Only @runtime protocols can be used with'
                                   ' instance and class checks', e)
+        if (isinstance(e.callee, RefExpr) and len(e.args) == 2 and
+                e.callee.fullname == 'builtins.issubclass'):
+            for expr in mypy.checker.flatten(e.args[1]):
```

Comment:
Move the body to a separate method, as this function is already quite long. Maybe also move the body of the above if statement (from line 269) to a separate method. You could also move the if statement to be nested under the above if statement (line 267). You'd only need the check `if e.callee.fullname == 'builtins.subclass'` which would simplify things slightly.